trending_data/src/main/webapp/init_map.js

```diff
@@ -81,6 +89,7 @@ function addMarkerToMapGivenInfo(countryName, countryCode, woeidCode, lat, lng,
   markerCluster.addMarker(marker);
 
   marker.addListener('click', () => {
+    map.setCenter({lat: lat, lng: lng});
```

Comment:
This is a pretty big change in behavior and is unrelated to making the map darker when the pop up opens. I am OK with keeping this in, but it should be documented somewhere that this PR is also making the map center on the markers when the user clicks on them.
Please edit the PR description before merging to reflect this.